_includes/footer.html

```diff
@@ -25,24 +25,33 @@
 			<section>
 				<a href="mailto:{{ site.email }}">
 						<div class="contact-method">
-							<span class="icon alt fa-envelope"></span>
+							<span class="icon alt fas fa-envelope fa-2x"></span>
 							<h3>Email | {{ site.email }}</h3>
 						</div>
 					</a>
 			</section>
 			<section>
 				<a href="{{ site.twitter_url }}">
 					<div class="contact-method">
+					<span class="icon alt fab fa-twitter fa-2x"></span>
 						<h3>Twitter | formulanano</h3>
-						<a href="{{ site.twitter_url }}" class="icon alt fa-twitter" target="_blank"></a>
 					</div>
-			</a>
+				</a>
 			</section>
+
 			<section>
 				<a href="{{ site.github_url }}">
 					<div class="contact-method">
+						<span class="icon alt fab fa-github fa-2x"></span>
 						<h3>GitHub | formulanano</h3>
-						<a href="{{ site.github_url }}" class="icon alt fa-github" target="_blank"></a>
```

Comment:
Same as above

Reply:
Well, this links are actually not functional because they are already in another anchor tag which does exactly same thing (ex. starts with line 42). Also they are used for icons but FA v5 doesn't support this usage I guess, icons are not shown. That span tags just work fine. Icons are shown and they are clickable as they are already in an anchor tag.